modules/ROOT/pages/tuning-pooling-profiles.adoc

```diff
@@ -0,0 +1,21 @@
+= Pooling Profiles
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Unlike singleton components, pooled components configure their component pool which contains multiple instances of the component to handle simultaneous incoming requests. Each pooled component has its pooling profile.
+
+== Use Pooling Profile Parameter
+Connectors such as Anypoint Connector for FTP (FTP Connector) or Anypoint Connector for SFTP (SFTP Connector) pool the connections by default and use a pooling profile to customize the connection pool. Add the `pooling-profile` parameter to the connectors when performance tests show it necessary.
```

Comment:
```suggestion
Connectors such as Anypoint Connector for FTP (FTP Connector) and Anypoint Connector for SFTP (SFTP Connector) pool connections by default and use a pooling profile to describe the connection pool. 
```
But this is confusing: "Add the `pooling-profile` parameter to the connectors when performance tests show it necessary."
If these connectors pool connections by default, WHY would a user need to add the profile?
And are we saying that adding this parameter actually creates the pool?